examples/model_load_store_lora/service.py

```diff
@@ -0,0 +1,63 @@
+import os
+import typing as t
+
+from diffusers import StableDiffusionImg2ImgPipeline
+from diffusers import StableDiffusionPipeline
+from PIL.Image import Image
+
+import bentoml
+
+sample_txt2img_input = dict(
+    prompt="pokemon with blue eyes",
+    height=768,
+    width=768,
+    num_inference_steps=50,
+    guidance_scale=7.5,
+    eta=0.0,
+)
```

Comment:
This input format is old for 1.1. Let's use the new format (just type annotate the function input arguments) and remove these lines.